.buildkite/commands/run-unit-tests.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash -eu
 
+APP=$1
+
 # Run this at the start to fail early if value not available
 echo '--- :test-analytics: Configuring Test Analytics'
 export BUILDKITE_ANALYTICS_TOKEN=$BUILDKITE_ANALYTICS_TOKEN_UNIT_TESTS
 
 echo "--- 📦 Downloading Build Artifacts"
-download_artifact build-products.tar
-tar -xf build-products.tar
+download_artifact build-products-$APP.tar
+tar -xf build-products-$APP.tar
```

Comment:
~~Given we don't have app-specific unit tests, this parameter could be considered unnecessary. We could apply YAGNI and hardcode `wordpress`.~~

~~🤔~~

~~At the same time, the implementation is quite lean and I guess it's handy to have the capability already. 😄~~

Actually, looking at the code below I see that there are other WordPress-specific values hardcoded (`name:WordPressUnitTests`), so really this script is only built for the WordPress unit tests. With that in mind, we might as well simplify it by removing input argument management and hardcode `build-products-wordpress.tar`.

Reply:
The same comment applies to the ui tests. As long as we only run them for Jetpack, we might as well remove the added complexity of the input parameter.

Reply:
yeah, i may have overthought/overcomplicated this, i agree with your assessment. updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20301/commits/5b78c495c4d8e052cf2220e4b6502dd981db943f